assopy/forms.py

```diff
@@ -146,87 +146,13 @@ class RefundItemForm(forms.Form):
         max_length=200,
         help_text=_("""Please enter the reason of your refund request"""),
         widget=forms.Textarea)
-    paypal = forms.EmailField(
-        label=_("Your paypal address"),
-        help_text=_("""If you prefer to receive payment via paypal"""),
-        required=False)
     bank = forms.CharField(
         label=_("Bank routing information"),
         help_text=_("""Please specify IBAN, BIC and bank address (if in Europe) or any needed information for a worldwide transfer"""),
-        required=False,
+        required=True,
```

Comment:
Why is it required now?

Reply:
Well, before the form supported bank transfer or paypal as the payment method, with both being optional, and the form's `clean` verified that at least one is present. With paypal removed as an option, the `bank` is the only remaining option, so it should be required and the `clean` can be removed.

Reply:
I'm not sure where (or if) this form is still used, but bank transfers are essentially equally deprecated. We don't support them at all, but the 'bank' payment type was in some places used to indicate the tickets that were bought with 100% vouchers (I guess as a legacy re-use "if there's no payment, default to bank"), so this can probably be removed as well. 

I think the followup to this is probably removal of the whole form anyway, since IIRC we don't support refunds either (I mean directly in the code, it's handled somehow by Finance WG)

cc @sm6xmm

Reply:
Good point about the refunds - the form does seem to be used in some legacy refund views, I wouldn't want to expand the scope of this PR to include them as well though; I'd rather keep the cleanup MRs pretty limited to make them easier to review, as there's many more coming.